src/SplitLargeFiles/FileProperties.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using ICSharpCode.SharpZipLib.GZip;
+
+namespace NuGet.Jobs.SplitLargeFiles
```

Comment:
This isn't a job. 

I'd suggest

```suggestion
namespace NuGet.Tools.SplitLargeFiles
```

Reply:
Sounds good.